packages/docgen/README.md

```diff
@@ -0,0 +1,250 @@
+# `docgen`
+
+`docgen` helps you to generate the _public API_ of your code. Given an entry point file, it outputs the ES6 export statements and their corresponding JSDoc comments in human-readable format.
+
+Some characteristics:
+
+* If the export statement doesn't contain any JSDoc, it'll look up for JSDoc up to the declaration.
```

Comment:
I'm not sure I understand this. Can you clarify?

Reply:
I guess you're saying if there's no JSDoc on the export, it will look for the JSDoc with the exported thing itself?

Reply:
Exactly. For example:

```js
/**
 * Adds two numbers.
 *
 * @param {number} term1 First number.
 * @param {number} term2 Second number.
 * @return {number} The result of adding the two numbers.
 */
function addition( term1, term2 ) {
	// Implementation would go here.
}

export default addition;
```

The export statement doesn't have any JSDoc, but the symbol declaration does.